app/views/audit_mailer/summary.html.erb

```diff
@@ -0,0 +1,30 @@
+<body style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1; padding: 10px 24px">
+<h2 style="font-weight: normal; text-transform: uppercase; font-size: 120%;"><%= _("<b>Foreman</b> Audit summary").html_safe %></h2>
+<ul style="list-style-type: none;">
+  <% if @audits.size > 0 %>
+    <%= _('Displaying %{num_audits} audits of %{total_audits} audits') % {:num_audits => @limit, :total_audits => @count} %>
+    <% @audits.each do |audit| %>
+      <li style="margin: 5px 15px; border-bottom: 1px solid #ccc; padding-bottom: 10px; width: 70%; font-size: 120%;">
+        <%
+           login = audit.user.login rescue nil
+           user_search_param = login ? "user = #{login}" : "username = \"#{audit.username}\""
+        %>
+        <b>
+          <%= link_to(audit.username.gsub(_('User'), ''), audits_path(:host => @url.host, :port => @url.port, :only_path => false, :protocol => @url.scheme, :search => user_search_param)) %>
```

Comment:
I know, I'm just copying the [current UI behavior ](https://github.com/theforeman/foreman/blob/6ace062d9dd9f0a529639d8123bf778072dce18e/app/helpers/audits_helper.rb#L82)